lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
@baer @exogen -- Can we just defer this whole conversation for now and remove

```js
context = context || this;
```

and allow a later `callback.apply(undefined|null|whatever)`?